client/layout/index.jsx

```diff
@@ -179,9 +178,6 @@ class Layout extends Component {
 				<QuerySites primaryAndRecent={ ! config.isEnabled( 'jetpack-cloud' ) } />
 				{ this.props.shouldQueryAllSites && <QuerySites allSites /> }
 				<QueryPreferences />
-				{ config.isEnabled( 'layout/query-selected-editor' ) && (
```

Comment:
Removes the `QuerySiteSelectedEditor` component that fetches editor preference on site change.